go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/cilium/fake v0.4.0
 	github.com/cilium/linters v0.0.0-20230711081823-012f25fa2197
 	github.com/cilium/lumberjack/v2 v2.3.0
-	github.com/cilium/proxy v0.0.0-20231202123106-38b645b854f3
+	github.com/cilium/proxy v0.0.0-20230911173141-49b6a7c9c075
```

Comment:
@sayboras this looks odd since we are downgrading the dependency.

Reply:
Good point, seems like we have bump the version in previous backport https://github.com/cilium/cilium/pull/29447